packages/storage/src/implementation/error.ts

```diff
@@ -261,6 +261,13 @@ export function noDownloadURL(): StorageError {
   );
 }
 
+export function missingPolyFill(polyFill: string): StorageError {
+  return new StorageError(
+    StorageErrorCode.UNSUPPORTED_ENVIRONMENT,
+    `${polyFill} is missing. Please install the right polyfills. Visit https://firebase.google.com/docs/web/environments-js-sdk#polyfills for more information.`
```

Comment:
Suggest "`${polyFill} is missing. Make sure to install the required polyfills. See..."